examples/multiple_tables_with_cursors/connector.py

```diff
@@ -0,0 +1,213 @@
+# This is an example for how to work with parent-child relationship between tables from incremental API endpoints.
+# It defines a simple `update` method, which upserts retrieved data to respective company and department tables.
+# THIS EXAMPLE IS THE JUST FOR UNDERSTANDING AND SIMULATES API CALL WITH STATIC HARD CODED DATA.
+# See the Technical Reference documentation
+# (https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update)
+# and the Best Practices documentation (https://fivetran.com/docs/connectors/connector-sdk/best-practices) for details
+
+# Import requests to make HTTP calls to API
+import requests as rq
+# Import required classes from fivetran_connector_sdk
+from fivetran_connector_sdk import Connector
+from fivetran_connector_sdk import Logging as log
+from fivetran_connector_sdk import Operations as op
+
+
+# Define the schema function which lets you configure the schema your connector delivers.
+# See the technical reference documentation for more details on the schema function:
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#schema
+# The schema function takes one parameter:
+# - configuration: a dictionary that holds the configuration settings for the connector.
+def schema(configuration: dict):
+    return [
+        {
+            "table": "company",
+            "primary_key": ["company_id"],
+            "columns": {
+                "company_id": "STRING",
+                "company_name": "STRING",
+                "updated_at": "UTC_DATETIME"
+            }
+        },
+        {
+            "table": "department",
+            "primary_key": ["department_id"],
+            "columns": {
+                "department_id": "STRING",
+                "company_id": "STRING",
+                "department_name": "STRING",
+                "updated_at": "UTC_DATETIME"
+            }
+        }
+    ]
+
+
+# Define the update function, which is a required function, and is called by Fivetran during each sync.
+# See the technical reference documentation for more details on the update function
+# https://fivetran.com/docs/connectors/connector-sdk/technical-reference#update
+# The function takes two parameters:
+# - configuration: dictionary contains any secrets or payloads you configure when deploying the connector
+# - state: a dictionary contains whatever state you have chosen to checkpoint during the prior sync
+# The state dictionary is empty for the first sync or for any full re-sync
+def update(configuration: dict, state: dict):
+    company_cursor = state["company_cursor"] if "company_cursor" in state else '0001-01-01T00:00:00Z'
```

Comment:
Can we add a comment here something like:
# setting cursor to correct format for initial sync and full resync cases

We might want to add this or a similar comment to some of our other examples also. - this was at the root of the  recent Arbonne issue - they weren't handling this 'empty' state case.